yt_dlp/extractor/boosty.py

```diff
@@ -162,9 +165,19 @@ def _extract_formats(self, player_urls, video_id):
 
     def _real_extract(self, url):
         user, post_id = self._match_valid_url(url).group('user', 'post_id')
+
+        auth_cookie = self._get_cookies('https://boosty.to/').get('auth')
+        auth_headers = {}
+        if auth_cookie is not None:
+            try:
+                auth_data = json.loads(urllib.parse.unquote(auth_cookie.value))
+                auth_headers['Authorization'] = f'Bearer {auth_data["accessToken"]}'
+            except (json.JSONDecodeError, KeyError):
+                self.report_warning('Failed to extract token from auth cookie.')
```

Comment:
Maybe include a bug report msg? You'd need to import `bug_reports_message` from utils
```suggestion
                self.report_warning(f'Failed to extract token from auth cookie{bug_reports_message()}')
```